ExploreASL_Initialize.m

```diff
@@ -697,111 +639,242 @@
 end
 
 
-
-%% ==================================================================================
 %% ==================================================================================
 function x = xASL_init_PrintCheckSettings(x)
-%xASL_init_PrintCheckSettings Check whether pre-defined settings existed in DATA_PAR.m
-% Prints these on the screen as the start of the pipeline.
-% Runs following steps:
-% 1) Set default settings if not defined
-% 2) Print data/study specific settings
-% 3) Print warnings
-
-%% -----------------------------------------------------------------------
-%% 1) Set default settings if not defined
-if ~isfield(x,'Quality') || (x.Quality~=0 && x.Quality~=1)
-    x.Quality = 1;
-    fprintf('%s\n', 'Default Quality=1 used (optimal quality)');
-end
-if ~isfield(x,'DELETETEMP') || (x.DELETETEMP~=0 && x.DELETETEMP~=1)
-    x.DELETETEMP = 1;
-%     fprintf('%s\n','Default x.DELETETEMP=1 used (delete files temporarily used for processing)');
-end
+    %xASL_init_PrintCheckSettings Check whether pre-defined settings existed in DATA_PAR.m
```

Comment:
Personal preference probably, but I don't see the benefit of starting a (sub-)function with a single indent, less will fit on the screen ;)

Reply:
Really? So in python you obviously have to do this, since this is how you write functions there. But I know this from C++ and JavaScript as well. There's even a wiki about this: https://en.wikipedia.org/wiki/Indentation_style I really like this style right here: Variant: 1TBS (OTBS), with this example:

```JavaScript
void checknegative(x) {
    if (x < 0) {
        puts("Negative");
    } else {
        nonnegative(x);
    }
}
```

Reply:
Yes, I like this as well in small functions. In longer functions, you don't really use this first indent visually anymore, so it just takes up space. Anyway, this seems indeed a good style to use!